azure-pipelines.yml

```diff
@@ -0,0 +1,158 @@
+  inputs:
+    solution: 'src/SFA.DAS.ProviderApprenticeshipsService.sln'
+    msbuildArgs: '/m'
+    platform: '$(buildPlatform)'
+    configuration: '$(buildConfiguration)'
+    clean: true
+
+- task: VSTest@2
+  displayName: 'Run NUnit Tests'
+  inputs:
+    testAssemblyVer2: |
+      **/bin/Release/*UnitTests.dll
+      **/bin/Release/*/*UnitTests.dll
+      !**/*ClientV2.UnitTests.dll
+    platform: '$(buildPlatform)'
+    configuration: '$(buildConfiguration)'    
+
+- task: NuGetCommand@2
+  displayName: 'NuGet Pack Provider Accounts API Client'
+  inputs:
+    command: 'pack'
+    packagesToPack: 'src/SFA.DAS.PAS.Account.Api.Client/SFA.DAS.PAS.Account.Api.Client.csproj'
+    packDestination: '$(Build.ArtifactStagingDirectory)/nuget-publish'
+    versioningScheme: byBuildNumber
+
+- task: VSBuild@1
+  displayName: 'Publish Provider Accounts API'
+  inputs:
+    solution: 'src/SFA.DAS.PAS.Account.Api/SFA.DAS.PAS.Account.Api.csproj'
+    vsVersion: "15.0"
+    msbuildArgs: '/p:DeployOnBuild=true /p:WebPublishMethod=Package /p:PackageAsSingleFile=true /p:SkipInvalidConfigurations=true /p:PackageLocation="$(build.artifactstagingdirectory)/Provider Accounts API"'
+    platform: 'anycpu'
+    configuration: '$(buildConfiguration)'
+    clean: true
+
+- task: VSBuild@1
+  displayName: 'Publish Provider Portal'
+  inputs:
+    solution: 'src/SFA.DAS.ProviderApprenticeshipsService.Web/SFA.DAS.ProviderApprenticeshipsService.Web.csproj'
+    vsVersion: "15.0"
+    msbuildArgs: '/p:DeployOnBuild=true /p:WebPublishMethod=Package /p:PackageAsSingleFile=true /p:SkipInvalidConfigurations=true /p:PackageLocation="$(build.artifactstagingdirectory)/Provider Portal"'
+    platform: 'anycpu'
+    configuration: '$(buildConfiguration)'
+    clean: true
+
+- task: VSBuild@1
+  displayName: 'Publish Provider WebJob'
+  inputs:
+    solution: 'src/SFA.DAS.PAS.Host.WebJob/SFA.DAS.PAS.Host.WebJob.csproj'
+    vsVersion: "15.0"
+    msbuildArgs: '/p:DeployOnBuild=true /p:WebPublishMethod=Package /p:PackageAsSingleFile=true /p:SkipInvalidConfigurations=true /p:PackageLocation="$(build.artifactstagingdirectory)/Provider WebJob"'
+    platform: 'anycpu'
+    configuration: '$(buildConfiguration)'
+    clean: true        
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet test'
+  inputs:
+    command: test
+    projects: |
+      src/SFA.DAS.ProviderUrlHelperTests/SFA.DAS.ProviderUrlHelperTests.csproj
+      src/SFA.DAS.PAS.Account.Api.ClientV2.UnitTests/SFA.DAS.PAS.Account.Api.ClientV2.UnitTests.csproj
+    testRunTitle: dotnet test run
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet build'
+  inputs:
+    projects: |
+      src/SFA.DAS.ProviderUrlHelper/SFA.DAS.ProviderUrlHelper.csproj
+      src/SFA.DAS.PAS.Account.Api.ClientV2/SFA.DAS.PAS.Account.Api.ClientV2.csproj
+      src/SFA.DAS.PAS.Account.Api.Types/SFA.DAS.PAS.Account.Api.Types.csproj
+    arguments: ' /p:AssemblyVersion=$(GitVersion.AssemblySemFileVer) /P:Configuration=$(buildConfiguration)'    
```

Comment:
Is the AssemblyVersion needed?

E.g. not in https://github.com/SkillsFundingAgency/das-platform-building-blocks/blob/master/azure-pipelines-templates/build/step/app-build.yml#L13